lib/far_mar/product.rb

```diff
@@ -0,0 +1,44 @@
+class FarMar::Product < FarMar::Base
+  attr_accessor :id, :name, :vendor_id
+  FILE = ("support/products.csv")
+
+  def initialize(attrs)
+   @id= attrs[0].to_i
+   @name = attrs[1]
+   @vendor_id = attrs[2].to_i
+  end
+
+
+  def vendor
```

Comment:
This method would be a great place to use the `FarMar::Vendor.find` method rather than `FarMar::Vendor.all`, the lines of code on 14 - 17 do exactly what the find method does.